src/DETHRACE/common/racesumm.c

```diff
@@ -707,9 +707,9 @@ void SpinWrecks(tU32 pFrame_period) {
 
     for (i = 0; i < gWreck_count; i++) {
         if (gWreck_array[i].customised == 0) {
-            BrMatrix34RotateY(&gWreck_array[i].rotation, BR_ANGLE_DEG(.05f * pFrame_period));
+            BrMatrix34RotateY(&gWreck_array[i].rotation, BrDegreeToAngle(0.05 * pFrame_period));
         }
-        BrVector3Copy(&translation, &gWreck_array[i].actor->t.t.translate.t);
+        memcpy(&translation, &gWreck_array[i].actor->t.t.translate.t, sizeof(br_vector3));
```

Comment:
Does it have to be a `memcpy`? Can it also an assignment?

Reply:
nice catch, yes, just tested, and same assembly generated. Will change these memcpys to assignments. I did think it was strange - like one of the engineers didn't know about `BrVector3Copy`